dev/run-tests.py

```diff
@@ -629,6 +647,7 @@ def main():
         # if we're on the Amplab Jenkins build servers setup variables
         # to reflect the environment settings
         build_tool = os.environ.get("AMPLAB_JENKINS_BUILD_TOOL", "sbt")
+        scala_version = os.environ.get("SCALA_PROFILE", "scala2.12")
```

Comment:
@dongjoon-hyun, since we're touching here shall we name and add `AMPLAP_JENKINS_BUILD_SCALA_PROFILE` at https://github.com/apache/spark/blob/master/dev/run-tests-jenkins.py#L177-L179 too just for consistency? Since the change is logically very simple, I think we won't bother have to test.

Reply:
(I was just double checking a similar commit like https://github.com/apache/spark/commit/4a73bed3180aeb79c92bb19aea2ac5a97899731a)

Reply:
Thanks! Got it.